src/Http/Controllers/Admin/TranslationsController.php

```diff
@@ -67,4 +74,222 @@ public function update(UpdateTranslation $request, Translation $translation)
         return redirect('admin/translation');
     }
 
+    /**
+     * @param UpdateTranslation $request
+     * @return \Symfony\Component\HttpFoundation\BinaryFileResponse
+     */
+    public function export(UpdateTranslation $request)
+    {
+        $currentTime = Carbon::now()->toDateTimeString();
+        $nameOfExportedFile = 'translations' . $currentTime . '.xlsx';
+        return Excel::download(new TranslationsExport($request), $nameOfExportedFile);
+    }
+
+
+    /**
+     * @param ImportTranslation $request
+     * @return array|\Illuminate\Http\JsonResponse|mixed
+     */
+    public function import(ImportTranslation $request) // FIXME create separate request class
+    {
+        if ($request->hasFile('fileImport')){
+            // FIXME extract all this code with helper private methods
+            $chooseLanguage = strtolower($request->importLanguage);
+            // let's grab only the first sheet
+
+            if($request->file('fileImport')->getClientOriginalExtension() != "xlsx"){
+                return response()->json("Unsupported file type", 409);
+            }
+
+            try{
```

Comment:
This part I would add to import service and return exception or validated collection.